modules/dailyhuntBidAdapter.js

```diff
@@ -0,0 +1,435 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import * as mediaTypes from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
```

Comment:
can you please update these * imports to just pull in the needed functions? We recently went through the project to remove all the 'utils.' root from bid adapters.

Reply:
imported only used functions from utils.